charts/victoria-metrics-k8s-stack/templates/servicemonitors/nodeexporter.yaml

```
@@ -12,6 +12,10 @@ spec:
 {{- if empty (index .Values "prometheus-node-exporter" "vmServiceScrape" "spec") }}
   endpoints:
   - port: metrics
+    metricRelabelings:
```

```suggestion
    metricRelabelConfigs:
```
metricRelabelings isn't supported by VMNodeScrape